CONTRIBUTING.adoc

```diff
@@ -50,15 +50,23 @@ If the project is _not_ intended to be used for production:
 * *MUST* clearly labeled in your repos README.adoc as `NOT INTENDED FOR PRODUCTION USE`
 
 
+=== Code Reviews
+Code reviews *MUST* be performed by at least one committer before a pull request can be merged.
 
-=== Code Contributions
-Contribution pull requests *MUST* contain the following:
+Reviewers *SHOULD* make suggestions in lieu of just comments, for issues found in pull request following github's guidelines https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request[here]
 
-* *Commit Description:* Properly formatted subject for the commit including a detailed description of the change. the subject line is pulled into the automatic release generation
+
+=== Pull Requests
+Pull requests *MUST* contain the following:
```

Comment:
I am not sure if I understand correctly what you want to achieve here. IMHO the things you describe below describe requirements for a *commit* of a PR. Or do you want to define some requirements for the PR comment in GitHub as well? If not, then IMHO we should rephrase this to make it clear that we are talking about *each individual commit* of a PR.

Reply:
@sophokles73 I rephrased the section so it is clearer, thanks for the feedback